README.md

```diff
@@ -1,2 +1,2 @@
 # application-api
-App Studio Application API
+App Studio Application API 
```

Comment:
Can this be switched back?

Reply:
It can be , but we (@dirgim ) agreed that it's not part of this PR , right ?

Reply:
There's still an added space there, I think @jsztuka is referring to that.